django/core/management/templates.py

```diff
@@ -277,16 +277,22 @@ def cleanup_url(url):
 
         if self.verbosity >= 2:
             self.stdout.write('Downloading %s' % display_url)
+
         try:
-            the_path, info = urlretrieve(url, os.path.join(tempdir, filename))
+            response = urlopen(Request(
+                url,
+                headers={'User-Agent': f'Django/{django.__version__}'}))
```

Comment:
I would use `get_version()` instead.

Reply:
Ack, changing

Reply:
Reverted to `django.__version__`, sorry.